esterni/piano-di-progetto/components/attualizzazione-dei-rischi.tex

```diff
@@ -1,5 +1,24 @@
 \documentclass[../piano-di-progetto]{subfiles}
 
 \begin{document}
+\subsection{Attualizzazione per la fase di analisi}%
+\label{sub:attualizzazione_fase_analisi}
+Nella tabella sottostante riportiamo i rischi incontrati nella fase di analisi, le modalità adottate per affrontarli e come proponiamo di gestire i suddetti rischi in futuro:
```

Comment:
" le modalità affrontate per affrontarli e come proponiamo di gestire i suddetti rischi in futuro" non è un po' una ripetizione

Reply:
sostituisco con utilizzate?